src/main/kotlin/me/zeroeightsix/kami/gui/text/VarMap.kt

```diff
@@ -147,7 +150,22 @@ object VarMap {
         "gametime_day_ticks" numeric { (mc.world?.timeOfDay?.toDouble() ?: 0.0) % 24000.0 },
         "realtime_ampm" string { ampmFormat.format(Date()) },
         "realtime_24h" string { militaryFormat.format(Date()) }
-    )
+    ).apply {
+        BaritoneIntegration {
+            put("baritone" string { BaritoneIntegration.recentControlProcess?.displayName() ?: "" })
```

Comment:
I would probably prefer something like `"None"` instead of `""` as a default value.

In, for example, KAMI's default HUD - this would make a lot more sense:
```
(white)None (grey)baritone
```
instead of just a "floating" `baritone`:
```
(white) (grey)baritone
       ^ space here!
```

Reply:
sorry for not replying to this. i completely overlooked it. but this is implemented now 👍